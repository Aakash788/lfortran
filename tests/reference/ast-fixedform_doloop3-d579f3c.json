{
    "basename": "ast-fixedform_doloop3-d579f3c",
    "cmd": "lfortran --fixed-form --show-ast --no-color {infile} -o {outfile}",
    "infile": "tests/fixedform_doloop3.f",
    "infile_hash": "57002b75b5c94260404c0a3a65d514c1629223773ae7b75e18b552ad",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast-fixedform_doloop3-d579f3c.stdout",
    "stdout_hash": "a2586fc09d58b420f13d87d283b8eb4dc2e21d72a634e53fb27c2eee",
    "stderr": "ast-fixedform_doloop3-d579f3c.stderr",
    "stderr_hash": "548ef49d41b7c9ead35244998499ef1eebb9beef166406e3bbd88a98",
    "returncode": 0
}