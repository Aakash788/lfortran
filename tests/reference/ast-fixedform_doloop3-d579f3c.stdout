(TranslationUnit [(Subroutine start1 [(n) (f) (l) (lout)] [] () () [] [] [] [(Declaration (AttrType TypeReal [] () None) [] [(f [(1 l DimensionExpr)] [] () None ()) (one [] [] 1 Equal ()) (two [] [] 2 Equal ())] ())] [(= 0 lout (+ 1 (/ n 2)) ()) (DoLoop 0 () 1 i 1 lout () [(= 1 (FuncCallOrArray f [] [(() i ())] [] []) two ())] () ()) (If 0 () (== (FuncCallOrArray mod [] [(() n ()) (() 2 ())] [] []) 0) [(= 0 (FuncCallOrArray f [] [(() lout ())] [] []) one ())] [] () () ()) (Return 0 () ())] [])])
